### ISSUE TITLE
Plugin options

### ISSUE THREAD

**Issue by goloveychuk** [cevek/ttypescript#16]
In docs I see that I can provide options for my transformer. How to do this, if there are 
https://github.com/cevek/ttypescript/blob/db3bec4fb6d86cf8992fa3f2cb8021149026fd4d/packages/ttypescript/src/PluginCreator.ts#L221-L224

**Comment by zerkalica** [comment]
Ttypescript passes config entry from tsconfig.json to second argument of plugin entry point.
```ts
function myTransformer(program: ts.Program, config?: PluginConfig) {
  config.someOption1 === 123
}
```

```json
{
    "compilerOptions": {
        "plugins": [
            { "transform": "transformer-module", "someOption1": 123, "someOption2": 321 }
        ]
    }
}
```

**Comment by goloveychuk** [comment]
yea, I have 
```json
"plugins": [
            { "transform": "./transformers/import-rename.ts", 
                "type": "config",
                "rename": {
                    "^(.*)\\.scss$": "$1.css"
                }
            },
        ]
```
config. And ttypescript raises error in console
`tsconfig.json plugins error: Unexpected property in plugins`
This is because of validating options function, which doesn't allow any option then `possibleKeys`
See the link to the code I provided

**Comment by cevek** [comment]
@goloveychuk Fixed in ttypescript@1.4.4
